web-ui/static/css/style_v1.css

```diff
@@ -0,0 +1,319 @@
+@tailwind base;

```

Comment:
Is it extracted from the code Asad gave us?

Reply:
Yes I have copied the generated css across.